### ISSUE TITLE
Auto fetching started more often than set in the settings

### ISSUE THREAD

**Issue by amal** [zielu/GitToolBox#34]
Seems like auto fetching is started much more often than set in the settings (15 minutes)

```
12:22:07.628: [test-project] git -c core.quotepath=false fetch translations --progress --prune
12:38:38.698: [test-project] git -c core.quotepath=false fetch origin --progress --prune
12:38:42.054: [test-project] git -c core.quotepath=false fetch translations --progress --prune
12:39:40.610: [test-project] git -c core.quotepath=false fetch origin --progress --prune
12:39:43.318: [test-project] git -c core.quotepath=false fetch translations --progress --prune
12:42:49.280: [test-project] git -c core.quotepath=false fetch translations --progress --prune
12:42:52.301: [test-project] git -c core.quotepath=false fetch origin --progress --prune
12:45:36.980: [test-project] git -c core.quotepath=false fetch translations --progress --prune
12:45:39.619: [test-project] git -c core.quotepath=false fetch origin --progress --prune
13:14:59.359: [test-project] git -c core.quotepath=false fetch origin --progress --prune
13:15:32.153: [test-project] git -c core.quotepath=false fetch translations --progress --prune
13:17:35.486: [test-project] git -c core.quotepath=false fetch origin --progress --prune
13:18:00.126: [test-project] git -c core.quotepath=false fetch translations --progress --prune
13:23:44.748: [test-project] git -c core.quotepath=false fetch origin --progress --prune
13:24:05.553: [test-project] git -c core.quotepath=false fetch translations --progress --prune
13:35:02.665: [test-project] git -c core.quotepath=false fetch translations --progress --prune
13:36:05.704: [test-project] git -c core.quotepath=false fetch origin --progress --prune
13:43:02.126: [test-project] git -c core.quotepath=false fetch translations --progress --prune
13:43:06.441: [test-project] git -c core.quotepath=false fetch origin --progress --prune
```

One more note:
Maybe auto fetching can be delayed when some heavy task, like application building, is running?
Sometimes each megabyte of RAM is needed for building and auto fetching launched at this very moment is not at all convenient :)

**Comment by zielu** [comment]
Similar to #33. Good idea to postpone during build as I already postpone fetch during indexing.

**Comment by zielu** [comment]
Fixed in 171.1 / 163.3
